doc/advanced.rst

```diff
@@ -383,3 +383,16 @@ the ``sphinx_gallery.gen_gallery`` extension, you can use in ``conf.py``::
     extensions = ['sphinx_gallery.load_style']
 
 This will only cause the ``gallery.css`` file to be added to your build.
+
+Add your own CSS styling
+========================
+
+To alter the appearance of your examples gallery you can add your own CSS file
+using the Sphinx configuration `html_static_path
+<https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_static_path>`_.
```

Comment:
Better to use an intersphinx link like
```suggestion
using the Sphinx configuration :ref:`sphinx:confval-html_static_path>`_.
```
This wasn't trivial because of https://github.com/sphinx-doc/sphinx/issues/5562 so I pushed a commit to do it